packages/app/src/app/overmind/namespaces/live/internalActions.ts

```diff
@@ -191,6 +191,8 @@ export const initializeModuleState = (
       moduleInfo,
     });
   });
+  // TODO: enable once we know exactly when we want to recover
+  // actions.files.internal.recoverFiles();
```

Comment:
I know it's silly to add back in a TODO comment, but it seemed the most sensible to just make it a complete revert of that PR, just in case.